static/js/graphs_page.js

```diff
@@ -2609,4 +2992,17 @@ var cytoscapeGraph = {
         });
     }
 
-};
\ No newline at end of file
+};
+
+//variables for cola
+//Different from other layout options, cola can choose options for other variables
+//Initial edge length set up to 45 in range of 1 to 200
+var cola_edge_length = 45;
```

Comment:
All of these variables need to go as parameter in colaOption or colaLayoutWidget module. Initialize them when the colaOption module's init function.

Reply:
I pushed the changes, and added cola.min.js file in static/components instead of calling Jeff's web url for cola.min.js